.github/workflows/deploy-cloud.yaml

```diff
@@ -20,6 +20,12 @@ jobs:
       APPS_MARKETPLACE_API_URI: "https://apps.saleor.io/api/v2/saleor-apps"
       IS_CLOUD_INSTANCE: true
     steps:
+      - name: Check region
+        if: ${{ !contains(fromJSON('["eu", "us"]'), env.REGION) }}
+        uses: actions/github-script@v3
```

Comment:
nodejs 12 is deprecated, use the latest version. But using an action is useless here, just use `echo 'something' >&2; exit 1`